webapp/src/app/chat/chat/chat.component.html

```diff
@@ -0,0 +1,28 @@
+@if (isLoading()) {
```

Comment:
In other components we try to go for Skeletons instead of Spinners since they are much easier on the eye in terms of layout shifts. 

Try to integrate Skeletons in here too. You can pass the "loading"-state down to the individual components, which can then handle it themselves.

Reply:
@GODrums 
Sure, was thinking of it too - the only problem in this case is that i am not sure what I need to create a Skeleton for in this case: we have one view which is shown if there are no sessions yet and a completely different one for the actual chat interface. What should I then make a Skeleton for when in is "undecided", which view to show (the number of user sessions is loading)

Reply:
Hm, I see your point here. I was a little confused about how the whole "fetching"-logic is fully contained in the `ChatComponent`.
I would have thought of it this way (inspired by the way https://chatgpt.com/ works):
- until the sessions are loaded, a spinner should be fine
- if existing sessions are found, they are displayed in the sidebar and the "chat-window" displays a message similar to "Please select a session or create a new one", basically a placeholder
- the user is now able to click on a session, replacing the placeholder with Skeletons of sample messages until the real messages are loaded

Maybe I'm overengineering this a little though, I think your approach is fine too!

Reply:
@GODrums good idea! I would make an extra issue for this frontend improvement and come back to it in another PR then 👍🏼